docs/apache-airflow/administration-and-deployment/logging-monitoring/logging-tasks.rst

```diff
@@ -57,34 +57,30 @@ In addition, you can supply a remote location to store current logs and backups.
 Writing to task logs from your code
 -----------------------------------
 
+Airflow uses standard Python `logging <https://docs.python.org/3/library/logging.html>`_ framework to
+write logs, and for duration of task, root logger is configured to write to task log.
+
 Most operators will write logs to the task log automatically. This is because they derive from they
 have a ``log`` logger that you can use to write to the task log.
```

Comment:
```suggestion
Most operators will write logs to the task log automatically. This is because they
have a ``log`` logger that you can use to write to the task log.
```